utils/remote.cc

```diff
@@ -2246,6 +2249,8 @@ static CURL* tr_curl_easy_init(struct evbuffer* writebuf)
     curl_easy_setopt(curl, CURLOPT_VERBOSE, debug);
     curl_easy_setopt(curl, CURLOPT_ENCODING, ""); /* "" tells curl to fill in the blanks with what it was compiled to support */
 
+    curl_easy_setopt(curl, CURLOPT_UNIX_SOCKET_PATH, unix_socket_path);
```

Comment:
What happens if this is set and the host isn't `localhost`? The docs are vague but did you try it? https://curl.se/libcurl/c/CURLOPT_UNIX_SOCKET_PATH.html

Reply:
Every url and ip gets whitelisted.
Authentication works.

Reply:
i think this is transmission-daemon's fault though.